manage.py

```diff
@@ -5,7 +5,7 @@
 import sys
 
 if __name__ == "__main__":
-    os.environ.setdefault("DJANGO_SETTINGS_MODULE", "edxval.settings")
+    os.environ.setdefault("DJANGO_SETTINGS_MODULE", "edxval.settings.base")
```

Comment:
how it was handling these settings before?

Reply:
there were no different settings file. Only settings.py. I created different settings file and base.py is same as settings.py